grunt/helpers.js

```diff
@@ -196,11 +196,30 @@ module.exports = function(grunt) {
         var includes = grunt.config('includes');
         var excludes = grunt.config('excludes');
 
+        var isIncluded;
+        var isExcluded;
+
         // carry on as normal if no includes/excludes
         if (!includes && !excludes) return true;
 
-        var isIncluded = includes && pluginPath.search(exports.getIncludedRegExp()) !== -1;
-        var isExcluded = excludes && pluginPath.search(exports.getExcludedRegExp()) !== -1;
+        // get plugins directory within theme/less
+        var compareVar = grunt.file.doesPathContain((grunt.config('sourcedir') + 'theme' + path.sep + grunt.config('theme') + path.sep + 'less' + path.sep + 'src' + path.sep + 'plugins'), pluginPath);
+
+        // if plugin directory within theme is in the filepath
+        if (compareVar) {
```

Comment:
'compareVar' isn't a very meaningful name, what about something like 'themeHasPlugin'?

Reply:
Yeah great recommendation. Is there a preference for name between @brian-learningpool's recommendation of `isThemePluginsFolder` and your's `themeHasPlugin`?

Reply:
Up to you! Either is good

Reply:
I prefer the prefix of the style is/has/was because it's immediately obvious that it's a boolean.